src/supervisor/watchers/handlers/pagination.ts

```diff
@@ -5,6 +5,7 @@ import type {
   KubernetesObject,
 } from '@kubernetes/client-node';
 
+import { trimItems } from './workload';
```

Comment:
where is this defined? I can' see it in the workload file

Reply:
https://github.com/snyk/kubernetes-monitor/pull/939/files#diff-497df571ce5ee886b45b30679d9f64cf189d762ca02bcfb2583178e2bfa23a8dR46